src/Microsoft.Health.Dicom.WebUtilities/Microsoft.Health.Dicom.WebUtilities.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
I don't really love `ImplicitUsings` as I think they hide information from developers, especially junior ones. I'd vote to remove this line